frontend/shared-components/state-select/UsStateSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import styles from "./state.module.css"
+
+export function USStateSelect() {
+    return (
+      <select className={styles.select}>
+        <option value="AL">AL</option>
```

Comment:
I still think we should iterate through a string array of states to generate these more DRY-ly!

Something like `{ states.map(state => <option key={state} valued={state}>{state}</option>) }` in place of all the options (with `states: string[] = ['AL', 'AK', 'AR', etc....]`)

EDIT: I only just saw your response, but couldn't add a reply outside of a github "review" - but the above is much DRYer because you're not writing the option element 50 times over (literally 😅) 
It doesn't matter that the array won't change, it can be a constant - the point is just that it's just so repetitive